### ISSUE TITLE
NPE (Columba-Deckard)

### ISSUE THREAD

**Issue by dmazinanian** [tsantalis/JDeodorant#32]
Clone # 248 (Group 317, Pair 1-2): org.columba.mail.folder.command.AddMessageCommand#execute (row 646) and org.columba.mail.folder.command.ExpungeFolderCommand#execute (row 647)

```
java.lang.NullPointerException
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.isExpressionUnderStatement(PreconditionExaminer.java:3155)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.isExpressionUnderStatement(PreconditionExaminer.java:3158)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.isExpressionUnderStatement(PreconditionExaminer.java:3158)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.isExpressionUnderStatement(PreconditionExaminer.java:3158)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.isExpressionUnderStatement(PreconditionExaminer.java:3158)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.isParameterizableExpression(PreconditionExaminer.java:3015)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.checkPreconditions(PreconditionExaminer.java:2042)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.checkCloneStructureNodeForPreconditions(PreconditionExaminer.java:2016)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.checkCloneStructureNodeForPreconditions(PreconditionExaminer.java:2018)
    at gr.uom.java.ast.decomposition.cfg.mapping.PreconditionExaminer.<init>(PreconditionExaminer.java:323)
    at gr.uom.java.ast.decomposition.cfg.mapping.PDGRegionSubTreeMapper.<init>(PDGRegionSubTreeMapper.java:40)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.getOptimalSolution(Application.java:1151)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.testRefactoring(Application.java:643)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.start(Application.java:184)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
    at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
```

**Comment by dmazinanian** [comment]
It seems that it also happens in JMeter-Deckard-Clone #412 (Group 522, Pair 1-6): org.apache.jmeter.functions.TestRegexFunction#testVariableExtraction (row 1031) and org.apache.jmeter.functions.TestTimeFunction#testYMD (row 1036)

**Comment by tsantalis** [comment]
Fixed in commit [a62becc07a3e9c8a3b65c0d846193cad3f443ad1](https://github.com/tsantalis/JDeodorant/commit/a62becc07a3e9c8a3b65c0d846193cad3f443ad1)
